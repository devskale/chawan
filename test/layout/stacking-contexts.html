<!DOCTYPE html>
<style>
div { position: relative; width: 4em; height: 4em; top: 2em; left: 2em }
</style>
<div style="background: red">
<div style="z-index: -1; background: green"></div>
</div>
<div style="background: blue"></div>
<br>
<div style="z-index: 0; background: red">
<div style="z-index: 1; background: green"></div>
</div>
<div style="background: blue"></div>
<br>
<div style="background: red">
<div style="z-index: 1; background: green"></div>
</div>
<div style="background: blue"></div>
