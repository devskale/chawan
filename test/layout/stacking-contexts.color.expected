

    [48;2;255;0;0m        [49m
    [48;2;255;0;0m        [49m
    [48;2;255;0;0m        [48;2;0;128;0m    [49m
    [48;2;255;0;0m        [48;2;0;128;0m    [49m
    [48;2;0;0;255m        [48;2;0;128;0m    [49m
    [48;2;0;0;255m        [48;2;0;128;0m    [49m
    [48;2;0;0;255m        [49m
    [48;2;0;0;255m        [49m

    [48;2;255;0;0m        [49m
    [48;2;255;0;0m        [49m
    [48;2;255;0;0m    [48;2;0;128;0m        [49m
    [48;2;255;0;0m    [48;2;0;128;0m        [49m
    [48;2;0;0;255m        [48;2;0;128;0m    [49m
    [48;2;0;0;255m        [48;2;0;128;0m    [49m
    [48;2;0;0;255m        [49m
    [48;2;0;0;255m        [49m

    [48;2;255;0;0m        [49m
    [48;2;255;0;0m        [49m
    [48;2;255;0;0m    [48;2;0;128;0m        [49m
    [48;2;255;0;0m    [48;2;0;128;0m        [49m
    [48;2;0;0;255m    [48;2;0;128;0m        [49m
    [48;2;0;0;255m    [48;2;0;128;0m        [49m
    [48;2;0;0;255m        [49m
    [48;2;0;0;255m        [49m
